pkg/hubble/metrics/flows-to-world/handler.go

```diff
@@ -53,7 +53,10 @@ func (h *flowsToWorldHandler) Init(registry *prometheus.Registry, options api.Op
 		Name:      "flows_to_world_total",
 		Help:      "Total number of flows to reserved:world",
 	}, labels)
-	h.worldLabel = fmt.Sprintf("%s:%s", pkglabels.LabelSourceReserved, pkglabels.IDNameWorld)
+	h.worldLabels = make(map[string]struct{})
```

Comment:
I didn't want to modify this logic too much. I don't know why this `worldLabels` solution was chosen, perhaps for no particular reason.  If there is another compelling reason to modify this PR then I will try to refactor this logic.

Reply:
Gotcha. Then maybe we can just move these labels to 3 global `var` and use a switch/if instead of a map? I think that aligns more closely with the previous logic, and avoids the map allocation.